lib/model/folder.go

```diff
@@ -50,8 +50,6 @@ func (f *folder) DelayScan(next time.Duration) {
 	f.scan.Delay(next)
 }
 
-func (f *folder) IndexUpdated() {}
-
 func (f *folder) IgnoresUpdated() {
```

Comment:
There should be a concrete implementation somewhere, as well as mocks in tests etc?

Reply:
This was literally the last remains of IndexUpdated.